cmd/zpool/zpool_main.c

```diff
@@ -7439,8 +7439,9 @@ main(int argc, char **argv)
 		 * it as such.
 		 */
 		char buf[16384];
+		memset(buf, 0, sizeof (buf));
 		int fd = open(ZFS_DEV, O_RDWR);
-		(void) strcpy((void *)buf, argv[2]);
+		(void) strncpy((void *)buf, argv[2], sizeof (buf));
```

Comment:
Using `strlcpy()` is the preferred fix for this because it guarantees the string is NULL terminated.  You can also remove the `memset()` above.